src/main/webapp/WEB-INF/views/order/orderDetails.jsp

```diff
@@ -0,0 +1,27 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Order details</title>
+</head>
+<body>
+<h1>Order details</h1>
+<table border="1">
+    <tr>
+        <th>Product</th>
+        <th>Price</th>
+    </tr>
+    <c:forEach var="product" items="${products}">
+        <tr>
+            <td>
+                <c:out value="${product.name}"/>
+            </td>
+            <td>
+                <c:out value="${product.price}"/>
+            </td>
+        </tr>
+    </c:forEach>
+</table>
+<a href="${pageContext.request.contextPath}">Back to the main page</a>
```

Comment:
here you can calculate the sum if you wish

Reply:
done